recipes/nbexamples/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set version = "0.2.0" %}
+{% set gh_org = "danielballan" %}
+{% set gh_repo = "nbexamples" %}
+
+package:
+  name: {{ gh_repo }}
+  version: {{ version }}
+
+source:
+  fn: {{ gh_repo }}-v{{ version }}.tar.gz
+  url: https://github.com/{{ gh_org }}/{{ gh_repo }}/archive/v{{ version }}.tar.gz
+  sha256: 96926c36333e57aee3820722f1107fbf6a3df9113b8585b14e572fccce385923
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - notebook >=4.2.0
+    - python
+    - setuptools
+  run:
+    - nbconvert
+    - nbformat
+    - notebook >=4.2.0
+    - python
+
+test:
+  imports:
+    - nbexamples
+
+about:
+  home: https://github.com/{{ gh_org }}/{{ gh_repo }}
+  license: BSD-3-Clause
+  summary: Conda environment and package access extension from within Jupyter
+
+extra:
+  recipe-maintainers:
+    - danielballan
+    - ericdill
+    - mariusvniekerk
+    - parente
```

Comment:
Just confirming everyone is ok with being listed as a maintainer.

cc @danielballan @mariusvniekerk @parente